README.md

```diff
@@ -113,16 +113,27 @@ Libraries for publishing to NPM should have their directory names prefixed with
 
 Apps should have their directory names prefixed with `app-`, e.g. `/project` becomes `/app-project`.
 
-## Production deployment
+## Deployment
 
-Deploys to production and staging are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/). Firstly, a base Docker image (`zooniverse/front-end-monorepo`) is built with the top-level Dockerfile, which installs and builds production versions of the `lib-` packages. 
+Deploys to production and staging are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/) using Docker images (descriptions found below). 
 
-That image is used as a base image to build `zooniverse/fe-project` and `zooniverse/fe-content-pages` images, from the Dockerfiles in `packages/app-project` and `packages/app-content-pages`. Those images run production (static) builds of the NextJS apps and are then deployed to Kubernetes.
+Merges to master deploy to a staging Kubernetes instance that uses Panoptes production. This is used for manual end to end behavior testing for new code and design reviews. `https://frontend.preview.zooniverse.org/projects/:project-owner/:project-name/` proxy redirects to the new NextJS app while the rest of sub-domain redirects to PFE. Staging projects can be loaded by adding this query param to the URL: `?env=staging`.
 
-The latest commit to master is deployed to https://frontend.preview.zooniverse.org
-The `production-release` git tag is deployed to https://www.zooniverse.org
+Deployments to a production Kubernetes instance are triggered by committing a `production-release` git tag on master. This can either be done using the git CLI or using the lita deploy command on slack. `https://www.zooniverse.org/projects/:project-owner/:project-name/classify` proxy redirects to the new NextJS app while the rest of the domain redirects to PFE. Currently the only project that is configured to do this is Planet Hunters TESS. Eventually more projects will migrate when they migrate to the new classifier.
```

Comment:
```suggestion
Commiting the `production-release` tag to master deploys to a production Kubernetes instance. This can either be done using the git CLI or using the lita deploy command on slack. `https://www.zooniverse.org/projects/:project-owner/:project-name/classify` proxy redirects to the new NextJS app while the rest of the domain redirects to PFE. Currently the only project that is configured to do this is Planet Hunters TESS. Eventually more projects will migrate when they migrate to the new classifier.
```
Changed to match the sense of the section about staging deploys.

Reply:
>using the git CLI or using the lita deploy command

Should we be explicit about how to do this? How much knowledge are we assuming about the person using these instructions?

Reply:
I'd prefer precise step by step instructions in a more secure place like the org's stackoverflow or how-to-zooniverse. The process on how this works at this point is the same as Panoptes, Caesar, etc. I'm not sure if we should make this public.